### DISCUSSION TITLE
Wireless USB Big Sur Adapter  Release V17 doesn't auto-load with Sonoma 14.4

### DISCUSSION THREAD

**Discussion by olekor** [chris1111/Wireless-USB-Big-Sur-Adapter#312] [Q&A]
After updating my Surface Pro 5 to Sonoma 14.4.1, I am encountering an issue where I need to reinstall the Wireless USB Big Sur Adapter every time I switch on the Wi-Fi (either EDIMAX EW-7611ULB or EDIMAX EW-7811DAC/AC600) as it does not start automatically. This was not an issue with Ventura. I have the latest Release V17, but it cannot install without disabling SIP, although it should work even with SIP enabled as shown in this guide.
 Additionally, in Privacy & Security settings, the Security tab does not show two options: "StatusBarApp was blocked....--Open Anyway"
 and "System software "Realtek.." has been updated --Allow".

**Comment by chris1111** [comment]
SIP must be disable to install the utility
Follow the guide correctly
